pkg/internal/feed/feed_action_timeline_item.go

```diff
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/MickStanciu/go-fn/fn"
```

Comment:
do we really need a dependency here? 

([A little copying is better than a little dependency.](https://go-proverbs.github.io/ ))

Reply:
we are using go-fn anyway. The de-dupe fn is in that lib